float64 newSpeedX
float64 newRotateZ
---
float64 respSpeedX
float64 respRotateZ